src/core/integrators/velocity_verlet_inline.hpp

```diff
@@ -0,0 +1,79 @@
+#ifndef INTEGRATORS_VELOCITY_VERLET_HPP
+#define INTEGRATORS_VELOCITY_VERLET_HPP
+
+#include "ParticleRange.hpp"
+#include "config.hpp"
+#include "particle_data.hpp"
+#include "rotation.hpp"
+
+inline void velocity_verlet_propagate_vel_pos(const ParticleRange &particles) {
```

Comment:
it would also be nice to have a docstring mentioning which part of VV is performed here (maybe a reference to literature)